config/webpack.config.js

```diff
@@ -34,7 +34,7 @@ const configGenerator = (options) => {
     filesToBuild = _.pick(entryFiles, options.entry.split(',').map(x => x.trim()));
   }
   filesToBuild.vendor = [
-    'core-js',
+    './src/js/common/polyfills',
```

Comment:
Gotcha. So these make it into the vendor bundle so we can use them in the other bundles. 👍

Thanks!